lib/matplotlib/patches.py

```diff
@@ -159,12 +159,13 @@ def contains_point(self, point, radius=None):
         Parameters
         ----------
         point : (float, float)
-            The point (x, y) to check in target coordinates of
-            ``self.get_transform()``. For patches added to a figure or axes,
-            these are display coordinates.
+            The point (x, y) to check; in target coordinates of
```

Comment:
This is not the place for a semicolon; a comma will suffice.
```suggestion
            The point (x, y) to check, in target coordinates of
```